.github/workflows/robustness-template.yaml

```diff
@@ -21,12 +21,22 @@ on:
         type: string
         default: "['ubuntu-latest']"
 permissions: read-all
+
 jobs:
   test:
     timeout-minutes: 210
     runs-on: ${{ fromJson(inputs.runs-on) }}
+    container:
```

Comment:
Running the `amd64` robustness workflows in containers does not appear to have negatively impacted runtime.

I ran workfows `on: pull_request` initially to validate. Workflows passed and runtimes were about the same:

- Original: https://github.com/etcd-io/etcd/actions/runs/5840581040
- This pr: https://github.com/etcd-io/etcd/actions/runs/5845356933